dev/io.openliberty.microprofile.openapi.3.1.internal_fat_tck/fat/src/io/openliberty/microprofile/openapi31/fat/tck/OpenAPITckTest.java

```diff
@@ -33,7 +37,17 @@
 @RunWith(FATRunner.class)
 public class OpenAPITckTest {
 
-    @Server("FATServer")
+    private static final String SERVER_NAME = "FATServer";
+
+    public static final FeatureSet MP50_OPENAPI_31 = MicroProfileActions.MP50.removeFeature("mpOpenAPI-3.0")
+                                                                             .addFeature("mpOpenAPI-3.1")
+                                                                             .build(MicroProfileActions.MP50_ID + "_mpOpenAPI-3.1");
+
+    // Run the TCK with EE9 features and EE10 features
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP60, MP50_OPENAPI_31);
```

Comment:
Ultimately mpOpenAPI-3.1 will only work with MP6, not MP5 ... might be a good idea to add a comment here to say that this is only temporary?